### ISSUE TITLE
Example style isn't applied in Chrome (but is applied in Incognito)

### ISSUE THREAD

**Issue by HartS** [EmilTholin/svelte-routing#203]
I have no idea why this is happening, but when I run the example app and navigate to `localhost:3000`, I get the unstyled app. In Incognito I get the styles.

In the dev console, I see the following message:

```
Resource interpreted as Stylesheet but transferred with MIME type text/html: "http://localhost:3000/bundle.css".
```

However, this appears in Incognito as well.

I've tried disabling the cache, but no luck.

**Comment by HartS** [comment]
Tried a few more things: using `127.0.0.1` instead of `localhost` allows styles to load correctly. Same with a domain that resolves to 127.0.0.1

**Comment by HartS** [comment]
More updates: This seems like a Chrome bug, or a problem with the way the service worker is caching `global.css` (even when `disable cache` is selected, I can see that the file is coming from the service worker; I navigated to the sources panel and took a look at `global.css` in a panel where styles were working and where they weren't. It turns out the contents are altogether different
